test/SlowTests/Issues/RavenDB-19932.cs

```diff
@@ -0,0 +1,29 @@
+using FastTests;
+using Raven.Server.Documents.Indexes.Persistence.Corax;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_19932 : RavenTestBase
+{
+    public RavenDB_19932(ITestOutputHelper output) : base(output)
+    {
+    }
+
+    [Fact]
+    public void GrowableHashSetForProjectionInCoraxIndexReadOperation()
+    {
+        var growableHashSet = new CoraxIndexReadOperation.GrowableHashSet<ulong>(maxSizePerCollection: 10);
+
+        for (int i = 0; i < 100; ++i)
+            growableHashSet.Add((ulong)i);
+        
+        Assert.True(growableHashSet.HasMultipleHashSets);
+
+        for (int i = 0; i < 100; ++i)
+            Assert.True(growableHashSet.Contains((ulong)i));
+
+        Assert.False(growableHashSet.Add(0));
```

Comment:
1, Let's verify that for all already added items it returns `false`. 
2. Can we introduce some random factor in the test? The items won't be so sequentially arranged under the covers then.

Reply:
Done